### PR TITLE
NXJS-92: add tests for UTF-8 blob filenames

### PR DESCRIPTION
**Pull request by troger**

### CODE REVIEW

File: test/utf8-filename.spec.js [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+        .input(nuxeoBlob)
+        .execute()
+        .then(() => nuxeo.repository().fetch(FILE_TEST_PATH))
+        .then((doc) => {
+          expect(doc.get('file:content').name).to.equal('café.txt');
+        });
+    });
+
+    it('with FileManager.Import operation', () => {
+      const nuxeoBinBlob = createTextBlob('foo', 'café.bin');
```
Comment (ataillefer) [inline test/utf8-filename.spec.js]: Why can't you use `nuxeoBlob` here?

File: test/utf8-filename.spec.js [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+        .context({ currentDocument: WS_JS_TESTS_PATH })
+        .input(nuxeoBinBlob)
+        .execute()
+        .then((doc) => {
+          expect(doc.get('dc:title')).to.equal('café.bin');
+        });
+    });
+  });
+
+  describe('should retrieve a blob UTF-8 filename', () => {
```
Comment (ataillefer) [inline test/utf8-filename.spec.js]: ... *with an* UTF-8 filename

File: test/utf8-filename.spec.js [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+    before(() => {
+      const nuxeoBlob = createTextBlob('foo', 'café.txt');
+      return nuxeo.operation('Blob.AttachOnDocument')
+        .param('document', FILE_TEST_PATH)
+        .input(nuxeoBlob)
+        .execute();
+    });
```
Comment (ataillefer) [inline test/utf8-filename.spec.js]: Shouldn't the following tests also be executed with a blob uploaded by batch upload and attached to a document with `/upload/{batchId}/execute`?

**Comment by ataillefer** [comment]
:+1:

File: test/utf8-filename.spec.js (hunk 2 above)
Comment (troger) [inline test/utf8-filename.spec.js]: I don't care about the blob itself, I just want to retrieve and test the filename

File: test/utf8-filename.spec.js (hunk 3 above)
Comment (troger) [inline test/utf8-filename.spec.js]: :+1:
Will add that.

File: test/utf8-filename.spec.js (hunk 1 above)
Comment (troger) [inline test/utf8-filename.spec.js]: I don't want to create a `Note` but a `File`.

**Pull request merged**